src/imitation/rewards/reward_net.py

```diff
@@ -221,50 +251,6 @@ def build_summaries(self):
         tf.summary.histogram("shaping_new", self._new_shaping_output)
 
 
-def build_basic_theta_network(
-    hid_sizes: Optional[Iterable[int]],
-    obs_input: Optional[tf.Tensor],
-    next_obs_input: Optional[tf.Tensor],
-    act_input: Optional[tf.Tensor],
-    **kwargs: dict,
-):
-    """Builds a reward network depending on specified observations and actions.
-
-    All specified inputs will be preprocessed and then concatenated. If all
-    inputs are specified, then it will be a :math:`R(o,a,o')` network.
-    Conversely, if `next_obs_input` and `act_input` are both set to `None`, it
-    will depend just on the current observation: :math:`R(o)`.
-
-    Arguments:
-      hid_sizes: Number of units at each hidden layer. Default is [], i.e. linear.
-      obs_input: Previous observation.
-      next_obs_input: Next observation.
-      act_input: Action.
-      **kwargs: Passed through to `util.build_mlp`.
-
-    Returns:
-      tf.Tensor: Predicted reward.
-
-    Raises:
-      ValueError: If all of obs_input, next_obs_input and act_input are None.
-    """
-    if hid_sizes is None:
-        hid_sizes = [32, 32]
-
-    with tf.variable_scope("theta"):
-        inputs = [obs_input, act_input, next_obs_input]
-        inputs = [x for x in inputs if x is not None]
-        if len(inputs) == 0:
-            raise ValueError("Must specify at least one input")
-
-        inputs = [tf.layers.flatten(x) for x in inputs]
-        inputs = tf.concat(inputs, axis=1)
-        theta_mlp = util.build_mlp(hid_sizes=hid_sizes, name="reward", **kwargs)
-        theta_output = util.sequential(inputs, theta_mlp)
-
-        return theta_output, theta_mlp
-
-
 class BasicRewardNet(RewardNet, serialize.LayersSerializable):
```

Comment:
In the doc string it says that we are using a linear function approximator, but I think the default right now is actually (32, 32). WE should either actually use linear function approx by passing in `hid_size=[]` or update the docstring.

Reply:
Good catch, fixed. I've also removed the bit about the AIRL paper since we no longer match that I guess.